src/nvim/eval.c

```diff
@@ -9595,13 +9595,15 @@ static void f_getcwd(typval_T *argvars, typval_T *rettv, FunPtr fptr)
       if (from) {
         break;
       }
-    case kCdScopeTab:         // FALLTHROUGH
+      // fallthrough
```

Comment:
I do not see a reason to shout here, fallthrough is not something which requires action. It is also not an abbreviation of some kind.